backend/app/routers/authentication.py

```diff
@@ -116,13 +117,20 @@ async def get_admin(
 
 
 @router.get("/users/me/admin_mode")
-async def get_admin_mode(current_username=Depends(get_current_user)) -> bool:
+async def get_admin_mode(
```

Comment:
1. Should we split the dependency from the route and put it in dependencies.py?
2. Should we rename `superadmin` to `admin` or `force_admin` or something else?

Reply:
I don't quite understand question 1. can you say more about "split the dependency from the route"?

Reply:
We need to rename `superadmin` to `force_admin`, since `admin` has been used in `admin=Depends(get_admin)`

Reply:
I don't think this was introduced in this PR, but fastapi dependencies don't need to be routes. More importantly this route returns just a boolean and not a JSON document, so we could separate the two and make the route return a valid JSON document. I can make the change.